ofl/solitreo/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,15 @@
+<p>
+This font was developed by the <a href="https://documentingjudeospanish.com/">Documenting Judeo-Spanish Project</a>.
+</p>
+<p>
+Documenting Judeo-Spanish is a digital humanities project that began in 2019 under the leadership of Dr. Bryan Kirschen. Recalling his initial fascination with Solitreo and the limited resources available to learn this script, Dr. Kirschen decided to focus this project on the cursive variety that was once common to speakers of Judeo-Spanish around the world. A nearly-extinct alphabet to an endangered language, this style of writing can be found in documents ranging from journal entries and ledgers to personal correspondence and community minutes. Many of these very documents are sitting in basements and attics today and, to the untrained eye, are mistaken for Hebrew.
+</p>
+<p>
+Judeo-Spanish (Ladino) refers to the variety of Spanish that developed among Jewish populations who were expelled from Spain in 1492 and subsequently settled throughout Turkey and the Balkans, then of the Ottoman Empire. These Jews, known as Sephardim, preserved many features of Medieval (varieties of) Spanish, while incorporating linguistic elements from the languages spoken in their surroundings, including: Turkish, Greek, Serbo-Croatian, French, Italian, and Arabic. As a Jewish language, the Spanish of the Sephardim has always been in contact with Hebrew. And while Judeo-Spanish may sound like other Romance languages, in writing, it would have traditionally appeared more similar to a Semitic language.
+</p>
+<p>
+Solitreo refers to the Hebrew-based cursive script once used by Sephardim; it is the cursive variety of the Rashi alphabet. Solitreo, or Soletreo, is derived from Galician/Portuguese, meaning ‘to spell.’ For many Sephardim, Solitreo was simply known as ganchos, meaning ‘hooks,’ due to the ligatures that form between letters. This style of writing is distinct from the Ashkenazi-based alphabet used for cursive Hebrew today, making documents in Solitreo undecipherable to the untrained eye. A similar style of writing can also be found in documents written in Judeo-Arabic. Today, Solitreo is a relic of the past, as most writers of the language utilize Roman characters.
+</p>
+<p>
+Most non-Hebrew glyphs in this font were forked from <a href="https://github.com/itfoundry/kalam">Kalam</a>.
+</p>
```

Comment:
Missing last sentence "To contribute, see <git url>"

Reply:
This is fixed in commit [322d0a0](https://github.com/google/fonts/commit/322d0a0ee603516498796dbacffa506e101ee832).